plasmapy/formulary/quantum.py

```diff
@@ -66,8 +77,8 @@ def deBroglie_wavelength(V: u.m / u.s, particle) -> u.m:
 
     Warns
     -----
-    ~astropy.units.UnitsWarning
-        If units are not provided, SI units are assumed
+    : ~astropy.units.UnitsWarning
+        If units are not provided, SI units are assumed.
 
     Notes
```

Comment:
Delete this notes section this the documentation is now in the docstring header.